sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -133,8 +134,21 @@ case class StaticInvoke(
   override def nullable: Boolean = needNullCheck || returnNullable
   override def children: Seq[Expression] = arguments
 
-  override def eval(input: InternalRow): Any =
-    throw new UnsupportedOperationException("Only code-generated evaluation is supported.")
+  override def eval(input: InternalRow): Any = {
+    if (staticObject == null) {
+      throw new RuntimeException("The static class cannot be null.")
+    }
+
+    val parmTypes = arguments.map(e =>
+      CallMethodViaReflection.typeMapping.getOrElse(e.dataType,
+        Seq(e.dataType.asInstanceOf[ObjectType].cls))(0))
+    val parms = arguments.map(e => e.eval(input).asInstanceOf[Object])
```

Comment:
We need null checks here for inputs? Also, can we add a common function in `InvokeLike` to handle input arguments for other `InvokeLike` eprs? (I mean the interpreted version of `InvokeLike.prepareArguments`).

Reply:
Yeah, it is good since this would be used in 'Invoke', too. After supporting other types, I will create an utility function.

To discuss design choice (reflection or method handle), I put earlier version.